src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -359,6 +360,92 @@ public void validateNoKeyPersonnelDuplicates(DataAccessRequestData darData) {
     }
   }
 
+  @VisibleForTesting
+  public void validatePersonnelInSameInstitution(User user, DataAccessRequestData darData) {
+    Institution submitterInstitution = user.getInstitution();
+    String piEmail = darData.getPiEmail();
+    String soEmail = darData.getSigningOfficialEmail();
+    String itEmail = darData.getItDirectorEmail();
+    List<String> collaboratorsEmails =
+        darData.getInternalCollaborators().stream().map(Collaborator::getEmail).toList();
+    List<String> labStaffEmails =
+        darData.getLabCollaborators().stream().map(Collaborator::getEmail).toList();
+
+    List<String> invalidMembers = new ArrayList<>();
+
+    if (emailDoesNotMatchInstitution(submitterInstitution, piEmail)) {
+      invalidMembers.add("Principal Investigator: " + piEmail);
+    }
```

Comment:
To avoid code duplication, I would make this method accept the type of item being checked and the list of errors to append to
```suggestion
    verifyEmailMatchesInstitution(submitterInstitution, piEmail, "Principal Investigator", invalidMembers);
```